hack/generate-cd.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SOURCE_PATH="$(dirname $0)/.."
+oci_images=$@
+REPO_CTX="eu.gcr.io/sap-se-gcr-k8s-private/cnudie/gardener/development"
+CA_PATH="$(mktemp -d)"
+BASE_DEFINITION_PATH="${CA_PATH}/component-descriptor.yaml"
+
+if [[ $EFFECTIVE_VERSION == "" ]]; then
+  EFFECTIVE_VERSION="$(${SOURCE_PATH}/hack/get-version.sh)"
+fi
+
+if ! which component-cli 1>/dev/null; then
+  echo -n "component-cli is required to generate the component descriptors"
+  echo -n "Trying to installing it..."
+  go get github.com/gardener/component-cli/cmd/component-cli
+
+  if ! which component-cli 1>/dev/null; then
+    echo -n "component-cli was successfully installed but the binary cannot be found"
+    echo -n "Try adding the \$GOPATH/bin to your \$PATH..."
+    go get github.com/gardener/component-cli/cmd/component-cli
+  fi
+fi
+if ! which jq 1>/dev/null; then
+  echo -n "jq canot be found"
+  exit 1
+fi
+
+echo "> Generate Component Descriptor ${EFFECTIVE_VERSION}"
+echo "> Creating base definition"
+component-cli ca create "${CA_PATH}" \
+    --component-name=github.com/gardener/landscaper/landscaper \
+    --component-version=${EFFECTIVE_VERSION} \
+    --repo-ctx=${REPO_CTX}
+
+# add oci images
+#for image in "${oci_images[@]}"
+#do
+#  echo "Adding ${image} to component descriptor"
+#  cat <<EOF | component-cli ca resources add "${CA_PATH}" -
+#name: ${image}
+#version: 'v0.0.1'
+#type: 'ociImage'
+#relation: 'external'
+#access:
+#  type: 'ociRegistry'
+#  imageReference: ${image}
+#EOF
+#done
+
+echo "> Creating ctf"
+CTF_PATH=./gen/ctf.tar
+mkdir -p ./gen
+rm ${CTF_PATH}
```

Comment:
if `./gen/ctf.tar` did not exist and with `set -e` set, the script will exit here
```suggestion
[ -e $CTF_PATH ] && rm ${CTF_PATH}
```